controllers/users/_preview_toolbar.htm

```diff
@@ -49,3 +49,20 @@
 </div>
 */
 ?>
+
+<?=
+    /**
+     * @event rainlab.user.extendPreviewToolbar
+     * Fires when preview user toolbar is rendered.
+     *
+     * Example usage:
+     *
+     *     Event::listen('rainlab.user.extendPreviewToolbar', function ($controller, $record) {
+     *         // return your HTML here
```

Comment:
Could you provide an example of markup that would make sense to include here?

Also what's the difference between the list toolbar and the preview toolbar?

Reply:
@LukeTowers I left it open as a comment because different developers may use different approaches. For example, I would use:
```
return $controller->makePartial('~/plugins/mohsin/acme/send_invoice_button.htm');
```
Another developer could use perhaps do it inline using something like:
```
return <<< BUTTON_INVOICE_HTML
<a
    href="<?= Backend::url('mohsin/acme/invoice/' . $record->id ?>)"
    class="btn btn-primary oc-icon-paper-plane">
    Resend Invoice
</a>
BUTTON_INVOICE_HTML;
```

Do you want me to add a particular approach? Kindly let me know and I'll make an additional commit for this.

Reply:
I would prefer recommending the makePartial approach. Also could you typehint the event parameters in the example? i.e. `(RainLab\User\Controllers\Users) $controller, (RainLab\User\Models\User) $record`

Reply:
Added.